tools/bismark/bismark_mapping/bismark_bowtie2_wrapper.xml

```diff
@@ -0,0 +1,561 @@
+<tool id="bismark_bowtie2" name="Bismark Mapper" version="0.10.2">
```

Comment:
Can you please change the tool version to reflect the bismark version, so 0.16.3 here?